src/main/java/core/mate/academy/model/TruckProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {

```

Comment:
Do not use raw types. Always parametrize interfaces and classes if it is possible. In this case, you have correctly parametrized the `MachineProducer` with the `Truck` type.